utilities/project_planning/calculate_available_weeks.py

```diff
@@ -0,0 +1,67 @@
+"""
+Calculate max weeks of programmer availability for yearly planning.
+
+Assumes the following:
+
+1. The number of maintainers stays the same
+2. We cannot anticipate availability changes like parental leave, part-time moves, etc.
+3. 5 weeks of holiday + sick leave with stipulation that this is just an estimate!!!!!
+4. Not all conferences are accounted for
+"""
+
+programmers_with_reduced_availability = {
+    # _ is part-time (safer to assume that isn't going to change)
+    "_": (52 / 2) * 0.6,
```

Comment:
```suggestion
    "_": 52 * 0.6,
```

Let's assume that `_` is full time for the full year, not just half.

Reply:
Full time or part time?

Reply:
Part time, sorry.